src/openforms/utils/translations.py

```diff
@@ -44,3 +46,17 @@ def __enter__(self):
         if current is not None:
             self.language = current
         return super().__enter__()
+
+
+def set_language_cookie(response: Response, language_code: str):
+    response.set_cookie(
+        key=settings.LANGUAGE_COOKIE_NAME,
+        value=language_code,
+        max_age=settings.LANGUAGE_COOKIE_AGE,
+        domain=settings.LANGUAGE_COOKIE_DOMAIN,
+        httponly=settings.LANGUAGE_COOKIE_HTTPONLY,
+        path=settings.LANGUAGE_COOKIE_PATH,
+        samesite=settings.LANGUAGE_COOKIE_SAMESITE,
+        secure=settings.LANGUAGE_COOKIE_SECURE,
+    )
+    return response
```

Comment:
```suggestion
def set_language_cookie(response: Response, language_code: str) -> None:
    response.set_cookie(
        key=settings.LANGUAGE_COOKIE_NAME,
        value=language_code,
        max_age=settings.LANGUAGE_COOKIE_AGE,
        domain=settings.LANGUAGE_COOKIE_DOMAIN,
        httponly=settings.LANGUAGE_COOKIE_HTTPONLY,
        path=settings.LANGUAGE_COOKIE_PATH,
        samesite=settings.LANGUAGE_COOKIE_SAMESITE,
        secure=settings.LANGUAGE_COOKIE_SECURE,
    )
```
I wouldn't return anything because that sort of gives the impression that this function does not have side-effects, but the `set_cookie` method only produces side effects.